### PR TITLE
Chinese localization & fix fields.md

### PR DESCRIPTION
**Pull request by sunny0826**

### CODE REVIEW

**Comment by zeusro** [comment]
哎🐴，之前翻译的内容竟然被删了，有点无语。

File: docs/zh/fields.md [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+```
+commonLabels:
+  someName: someValue
+  owner: alice
+  app: bingo
+```
+
+### commonAnnotations
+
+为所有资源增加 Annotions ，和 labels 一样是 key:value 的键值对。
```
Comment (Liujingfang1) [inline docs/zh/fields.md]: Annotions --> Annotations

File: docs/zh/fields.md [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+现有可定制对象。
+
+| 字段 | 类型 | 说明 |
+| --- | --- | --- |
+|[resources](#resources) | list | 包含 k8s API 对象的文件，或其他包含 kustomizations 文件的目录。 |
+|[CRDs](#crds)| list | CDR 文件，以允许在资源列表中指定自定义资源。 |
+
+## Generators
+
+当前及随后可以创建的对象。
```
Comment (Liujingfang1) [inline docs/zh/fields.md]: 当前及随后可以创建的对象 --》 生成可定制的对象

File: docs/zh/fields.md [hunk 3]
```diff
@@ -0,0 +1,436 @@
...
+| 字段 | 类型 | 说明 |
+| --- | --- | --- |
+|[configMapGenerator](#configmapgenerator)| list  | 列表中的每个条目都将创建一个 ConfigMap （它是n个 ConfigMap 的生成器）。 |
+|[secretGenerator](#secretgenerator)| list  | 此列表中的每个条目都将创建一个 Secret 资源（它是n个 secrets 的生成器）。 |
+|[generatorOptions](#generatoroptions)| string | generatorOptions 可以修改所有 ConfigMapGenerator 和 SecretGenerator 的行为。 |
+|[generators](#generators)| list | [插件](../plugins)配置文件。 |
+
+## Transformers
+
+What transformations (customizations) should be applied?
```
Comment (Liujingfang1) [inline docs/zh/fields.md]: What transformations (customizations) should be applied?  --》
可用的变换 或者其他更贴切的意思

File: docs/zh/fields.md [hunk 4]
```diff
@@ -0,0 +1,436 @@
...
+
+| 字段 | 类型 | 说明 |
+| --- | --- | --- |
+| [commonLabels](#commonlabels) | string | 为所有资源和 selectors 增加 Labels 。 |
+| [commonAnnotations](#commonannotations) | string | 为所有资源增加 Annotions 。 |
```
Comment (Liujingfang1) [inline docs/zh/fields.md]: Annotions --》 Annotations

File: docs/zh/fields.md [hunk 5]
```diff
@@ -0,0 +1,436 @@
...
+| [images](#images) | list | 修改镜像的名称、tag 或 image digest ，而无需使用 patches 。 |
+| [inventory](#inventory) | struct | 指定注释将包含构建结果摘要的对象。Specify an object who's annotations will contain a build result summary. |
```
Comment (Liujingfang1) [inline docs/zh/fields.md]: 指定注释将包含构建结果摘要的对象。Specify an object who's annotations will contain a build result summary.  -》 用于生成一个包含清单信息的对象

File: docs/zh/fields.md [hunk 6]
```diff
@@ -0,0 +1,436 @@
...
+| [patchesJson6902](#patchesjson6902) | list | 列表中的每个条目都应可以解析为 kubernetes 对象和将应用于该对象的 JSON patch 。 |
+| [transformers](#transformers) | list | [插件](../plugins)配置文件。 |
+
+## Meta
+
+[k8s metadata]: https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields
+
+| 字段 | 类型 | 说明 |
+| --- | --- | --- |
+| [vars](#vars) | string | var 是包含该对象的变量名、对象引用和字段引用的元组。 |
```
Comment (Liujingfang1) [inline docs/zh/fields.md]: var 是包含该对象的变量名、对象引用和字段引用的元组 --》 获取一个对象中的字段并插入到另外的对象中

**Comment by Liujingfang1** [comment]
I don't see the field of `configurations` which is used to customize the path field for transformers. Can you check if it is present in the original file?

File: docs/zh/fields.md (hunk 3 above)
Comment (sunny0826) [reply docs/zh/fields.md]: 这里想了好几个感觉都不合适，就暂时没有翻译

**Comment by sunny0826** [comment]
> I don't see the field of `configurations` which is used to customize the path field for transformers. Can you check if it is present in the original file?

I can't find the field of `configurations` in `fields.md` or `kustomization.yaml`. The `fields.md` file may need to be updated.

**Comment by Liujingfang1** [comment]
> > I don't see the field of configurations which is used to customize the path field for transformers. Can you check if it is present in the original file?

> I can't find the field of configurations in fields.md or kustomization.yaml. The fields.md file may need to be updated.

OK. I'll update it.

**Pull request merged**